src/service/core-services.js

```diff
@@ -80,6 +82,11 @@ export function installRuntimeServices(global) {
 export function installAmpdocServices(ampdoc) {
   const isEmbedded = !!ampdoc.getParent();
 
+  if (RUNTIME3) {
+    installCustomElements(ampdoc);
```

Comment:
FMI: all the code that uses owners would crash right?

Reply:
I did it here on purpose to find the usages faster in this POC. We, of course, can't do that in refactoring. But we can deploy a no-op owner service via an interface for our experiment. Alternative would be to keep both services functional, but to get close enough where we can remove owners service and all of its uses as soon as we're ready. This is exactly a kind of tactical point I'd like to discuss for planning this migration.